src/app.config.ts

```diff
@@ -25,6 +25,9 @@ export abstract class AbstractAppConfig {
   abstract getPrdUrl(): string;
   abstract getCacheTimeOut(): number;
   abstract getWorkAllocationApiUrl(): string;
+  getUserInfoApiUrl(): string {
+    return undefined;
+  }
```

Comment:
Do you mind adding a comment why its undefined?

Reply:
It is undefined so it does not cause breaking change on master webapp so that when it is pushed prior to workallocation-6 there are no issues

Reply:
@udaydenduluri33, this is specifically not an `abstract` method so it doesn't count as a breaking change.